### ISSUE TITLE
[Accessibility checks] Add accessible title to iframe on events page

### ISSUE THREAD

**Issue by rivaquiroga** [r-devel/r-devel.github.io#27]
The Teamup calendar iframe on the events page (`events/index.qmd`, line 23) is missing a `title`  attribute, which axe-core flags as a "serious" accessibility violation (`frame-title` rule: screen reader users can't tell what the embedded frame is).

The fix is to add a descriptive `title` attribute to the iframe, e.g.:
  
<iframe src="https://teamup.com/..." title="R Contributor events calendar" ...></iframe>

**Comment by soph-scott** [comment]
I'll work on this one!
